dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/FeatureResolverWrapper.java

```diff
@@ -318,6 +325,28 @@ public static RequiredFeatureMap getAllRequiredFeatures(WebSphereRuntime wr, Lis
                 fe.getRequiredFeatures(wr, supportedModules, supportedDeltas, existingFeatures, featureMap, includeAll, monitor);
             }
         }
+
+        // Replace features in the map with ones that specify their alternatives from the override list
+        for (int i = 0; i < featureMap.getFeatures().length; i++) {
+            FeatureResolverFeature feature = featureMap.getFeatures()[i];
+            for (String key : alternativeFeatureMap.keySet()) {
+                if (feature.name.startsWith(key)) {
```

Comment:
When matching the key should it be more specific so that it does not match anything that starts with the key which includes jsfContainer as well.  I know that this would not cause any problems now but thinking more about the future if new features are added.

   feature.name.equals(key) || feature.name.startsWith(key + "-")

Reply:
Good catch! Added